src/Microsoft.OpenApi/Models/OpenApiRequestBody.cs

```diff
@@ -191,7 +190,7 @@ internal OpenApiBodyParameter ConvertToBodyParameter()
             };
             if (bodyParameter.Extensions.ContainsKey(OpenApiConstants.BodyName))
             {
-                bodyParameter.Name = (Extensions[OpenApiConstants.BodyName] as OpenApiString)?.Value ?? "body";
+                bodyParameter.Name = (Extensions[OpenApiConstants.BodyName].ToString()) ?? "body";
```

Comment:
avoid null coalesce with strings since it doesn't take care of empty strings. use type assertion + string.IsNullOrEmpty instead